src/warnet/warnet.py

```diff
@@ -91,9 +92,9 @@ def _warnet_dict_representation(self) -> dict:
     @classmethod
     @bubble_exception_str
     def from_graph_file(
-        cls, base64_graph: str, config_dir: Path, network: str = "warnet"
```

Comment:
IMO if anything the formatting change here should be:

```python
    def from_graph_file(
        cls,
        base64_graph: str,
        config_dir: Path,
        network: str = "warnet",
        backend: str = "compose",
    ):
```

But I think after we merge I am going to run `ruff` over the entire codebase quickly before any followups are started, to unify everything.